src/Designer/frontend/packages/shared/src/types/QueryKey.ts

```diff
@@ -36,6 +36,8 @@ export enum QueryKey {
   OrgCodeLists = 'OrgCodeLists',
   OrgCodeListsNew = 'OrgCodeListsNew',
   OrgTextLanguages = 'OrgTextLanguages',
+  GetSharedResourcesByPath = 'GetSharedResourcesByPath',
+  GetSharedResourcesByPathAndReference = 'GetSharedResourcesByPathAndReference',
```

Comment:
Vi pleier ikke ha med `Get` i nøkkelen, bare hva slags data den gjelder for. Alt kommer jo fra `GET`-spørringer.